src/java/picard/sam/SortSam.java

```diff
@@ -51,16 +51,27 @@
 )
 public class SortSam extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Sorts a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool sorts the input SAM or BAM file by coordinate, queryname  or some other property " +
-            "of the SAMRecord. Input and output formats (SAM or BAM) are determined by the file extension." +
-            "<br />" +
+    static final String USAGE_DETAILS = "This tool sorts the input SAM or BAM file by coordinate, queryname (QNAME), or some other property " +
+            "of the SAM record. The SortOrder of a SAM/BAM file is found in the SAM file header tag (@HD) in the field labeled (SO).  " +
+            "" +
+            "<p>For a coordinate sorted SAM/BAM file, read alignments are sorted first by the reference sequence name (RNAME) field using the " +
+            "reference sequence dictionary (@SQ tag).  Alignments within these subgroups are secondarily sorted using the left-most mapping " +
+            "position of the read (POS).  Subsequent to this sorting scheme, alignments are listed arbitrarily.</p>" +
+            "" +
+            "For QNAME-sorted alignments, all alignments are grouped using the query name (QNAME) field but the alignments are not necessarily sorted within these groups.  " +
+            "Reads having the same QNAME are derived from the same template.  " +
```

Comment:
Would prefer to use the fully spelled out word. QNAME is the name of a field in a SAM, not the thing itself.